pkg/gui/controllers/helpers/refresh_helper.go

```diff
@@ -373,6 +373,12 @@ func (self *RefreshHelper) refreshBranches() {
 		self.c.Log.Error(err)
 	}
 
+	// Need to refresh the commits view because the visualization of local
+	// branch heads might have changed
+	if err := self.c.PostRefreshUpdate(self.c.Contexts().LocalCommits); err != nil {
+		self.c.Log.Error(err)
+	}
```

Comment:
This is a problem. It causes the main panel to flicker whenever we refresh branches; for example, this happens for every background fetch, every 60s. Sometimes this even causes the main view to scroll back to the top, which is super annoying when you're in the middle of reviewing a branch. Not always though, sometimes it just flickers, which is bad enough though.

@jesseduffield Any suggestions how to do this differently so that it doesn't have this problem?

Reply:
No it doesn't (neither on master nor in this branch). But this is a refresh of the local commits view, so it's not surprising that this doesn't flicker when the branches panel is focused.

Reply:
The solution is to just call `HandleRender` instead of `PostRefreshUpdate`; see f1dd64e24378.